site-src/implementations/v1.1.md

```diff
@@ -16,20 +16,21 @@ Implementations only appear in this page if they pass Core conformance for the r
-| traefik        | traefik                       | v3.1.0             | :white_check_mark: | :white_check_mark:     | :white_check_mark:      | :x:                      | :x:                                   | :white_check_mark:        | :white_check_mark:        | :white_check_mark:     | :white_check_mark:            | :x:                            | :x:                      | :x:                           | :x:                                 | :x:                                         | :x:                       | :x:                      | :white_check_mark:      | :x:                               | :x:                       |
+| Organization   | Project                       | Version            | GatewayPort8080    | HTTPRouteHostRewrite   | HTTPRoutePathRedirect   | HTTPRouteRequestMirror   | HTTPRouteResponseHeaderModification   | HTTPRouteSchemeRedirect   | HTTPRouteMethodMatching   | HTTPRoutePathRewrite   | HTTPRouteQueryParamMatching   | HTTPRouteParentRefPort   | GatewayHTTPListenerIsolation   | GatewayStaticAddresses   | HTTPRouteBackendProtocolH2C   | HTTPRouteBackendProtocolWebSocket   | HTTPRouteBackendRequestHeaderModification   | HTTPRouteBackendTimeout   | HTTPRoutePortRedirect   | HTTPRouteRequestMultipleMirrors   | HTTPRouteRequestTimeout   |
+|:---------------|:------------------------------|:-------------------|:-------------------|:-----------------------|:------------------------|:-------------------------|:--------------------------------------|:--------------------------|:--------------------------|:-----------------------|:------------------------------|:-------------------------|:-------------------------------|:-------------------------|:------------------------------|:------------------------------------|:--------------------------------------------|:--------------------------|:------------------------|:----------------------------------|:--------------------------|
+| GKE            | gke-gateway                   | 1.30.3-gke.1211000 | :white_check_mark: | :white_check_mark:     | :white_check_mark:      | :white_check_mark:       | :white_check_mark:                    | :white_check_mark:        | :x:                       | :x:                    | :x:                           | :x:                      | :x:                            | :x:                      | :x:                           | :x:                                 | :x:                                         | :x:                       | :x:                     | :x:                               | :x:                       |
```

Comment:
seeing 3 identical entries for GKE's `1.30.3-gke.1211000` version

Reply:
Good catch - I think we should probably include "mode" from the conformance reports in this table. In this case, GKE is reporting separately for 3 different underlying LBs:

https://github.com/kubernetes-sigs/gateway-api/blob/f5262011c93037166cf967c40c0e2651fa15d842/conformance/reports/v1.1.0/gke-gateway/standard-1.30.3-gxlb-report.yaml#L13

Reply:
good catch - didnt realize this was in the conformance reports. cmiiw, but this appears only in v1.1 and later? I changed the generate report script to reflect that!